### ISSUE TITLE
Notifications do not show achieved yields

### ISSUE THREAD

**Issue by PerfectDark99** [LoneGazebo/Community-Patch-DLL#6755]
1. Mod version (i.e Date - 4/17): 6/19

2. Mod list (if using Vox Populi only, leave blank):
- More Luxuries
- Speedy Settler (first settler has 10 move points)
- Really advanced setup
- Quick Turns
- Faster aircraft animations

3. Error description:

Notifications are not displaying the made yields say when borders expand or you kill a unit.
It shows the message + a colon eg. Winning battle...:   and then empty, no info on yields as it was before.
(Yields show for example over capital and are applied just not displayed in the notifications)

This is the second game with this version. The first game stopped working around turn 70. It simply crashes to desktop. Since I have the same issue with the yields not showing I fear this game at some point will also be corrupted and crash,

4. Steps to reproduce (optional):_

Get a policy or believe that gives yields and triggers a notification. Check if notification mentions the yields made.

**Comment by RecursiveVision** [comment]
Intended change by @ilteroi. A summary of yields obtained in the previous turn can be obtained by mousing over the instant yields "star" on the top panel.

**Comment by ilteroi** [comment]
maybe we should revert this

**Comment by RecursiveVision** [comment]
It seems to be unpopular.

**Comment by PerfectDark99** [comment]
I opened the issue because I thought it was a bug. While I prefer to have the yields shown, knowing that it it not a bug is good enough for me. 

Note that I thought it was a bug because I'm having issues with crash to desktop, At some point the game seems to be corrupted or trigger and action that leads to a crash. I thought it was related. SInce all games I started have been suffering from this and many other seem to have CTD issues I think something with this release is seriously problematic. I have never before had CTD with vox populi.
